zerver/openapi/zulip.yaml

```diff
@@ -12180,6 +12185,156 @@ paths:
                         description: |
                           An example JSON error response for when the user doesn't have permission
                           to subscribe other users to streams and `stream_ids` is not empty.
+
+  /invites/multiuse/{invite_id}:
+    patch:
+      operationId: edit-invite-link
+      summary: Edit a reusable invitation link
+      tags: ["invites"]
+      description: |
+        [Edit a reusable invitation link](/help/invite-new-users#edit-a-reusable-invitation-link)
+        which can be used to invite new users to the organization.
```

Comment:
I think you should mention the fields that can be changed by using this endpoint.